Assets/MRTK/Core/Utilities/WebRequestRest/Rest.cs

```diff
@@ -239,6 +234,8 @@ private static async Task<Response> ProcessRequestAsync(UnityWebRequest webReque
                 }
             }
 
+            webRequest.certificateHandler = certificateHandler;
+            webRequest.disposeCertificateHandlerOnDispose = false;
```

Comment:
Since this value defaults to true (i.e. disposeCertificateHandlerOnDispose = true by default), callers of the various functions here might expect that all of the function above (i.e. ones that use ProcessRequestAsync) actually dispose of the certificate handler for them - i.e. they're used to this functionality by default because of how UnityWebRequest usually works). Whether or not this assumption is true (i.e. I could very well be wrong here), it's probably still worth calling out explicitly (i.e. adding a <remarks></remarks> section to all of functions above like DeleteAsync, PutAsync, etc) that the certificate handler will not be disposed of after the web request is made (and that if you need it to be disposed, you - i.e. the caller - will need to dispose of it)

Reply:
Done by adding optional `disposeCertificateHandlerOnDispose` parameter.